tests/components/econet/test_config_flow.py

```diff
@@ -0,0 +1,100 @@
+"""Tests for the Econet component."""
```

Comment:
Please test the abort case with an existing entry with same email. Add a `MockConfigEntry` with that data before starting the flow.

Reply:
Added.